ns1/rest/monitoring.py

```diff
@@ -67,6 +67,18 @@ def delete(self, jobid, callback=None, errback=None):
             errback=errback,
         )
 
+    @property
+    def jobtypes(self, callback=None, errback=None):
+        return self._make_request(
+            "GET", "monitoring/jobtypes", callback=callback, errback=errback,
+        )
+
+    @property
+    def regions(self, callback=None, errback=None):
+        return self._make_request(
+            "GET", "monitoring/regions", callback=callback, errback=errback,
+        )
+
```

Comment:
waffling on whether I like these as properties:

Pro:
* read-only
* tidy conceptual separation from the basic CRUD ops

Con:
* kinda hate properties that do work (e.g. a db lookup or an http request), cause it can make it easy to miss, and easy to write non-performant code.

Reply:
I'm not opposed to this but don't have the Py expertise to have a strong opinion

Reply:
I saw it both ways and couldn't make a clear call on this so I asked @jwilner for feedback.  He reiterated your concern that putting this in a property makes it non-obvious that it's doing work.

At the end of the day, these are really still CRUD ops (the R specifically) and I think putting these in clearly named `load` functions is probably the least surprising way to implement this.

Reply:
cool, I'll make 'em functions, disambiguating them from the "standard" CRUD ops on the class isn't worth it, and leaves more options for non-breaking changes.

I'm kind of opposed to making top-level `load*` functions for these, but I'll take that discussion elsewhere